.github/workflows/publish.yml

```diff
@@ -0,0 +1,35 @@
+name: brew pr-pull
+on:
+  pull_request_target:
+    types:
+      - labeled
+jobs:
+  pr-pull:
+    if: contains(github.event.pull_request.labels.*.name, 'pr-pull')
```

Comment:
Is this something that anyone (especially outside of the org) can trigger with a PR?

Reply:
You have to be a maintainer to label, so it should be just fine